Add unit tests

GOVINDDIXIT opened the pull request:
Fix #638

GOVINDDIXIT committed:
add unit tests for widechat

ear-dev commented:
@GOVINDDIXIT There are circleCI changes needed to run these for every PR, correct?  Can you include them here please?

GOVINDDIXIT committed:
update circle ci config

GOVINDDIXIT commented:
@ear-dev Currently the unit tests are running under code analysis job for all PRs. I have renamed it to run-unit-tests and removed some unnecessary code from it. 👍

GOVINDDIXIT commented:
@jaytat0 Is anything more to be done here?

GOVINDDIXIT commented:
Base Project is Archived
